Source/Core/Core/NetPlayClient.cpp

```diff
@@ -5,6 +5,7 @@
 #include "Core/NetPlayClient.h"
 
 #include <algorithm>
+#include <cassert>
```

Comment:
Why not use our assert (``Common/Assert.h``)?

Reply:
Doesn't really matter either way, but OK.

Reply:
Forgot to reply saying it's been done, sorry.